src/core-scrabble.js

```diff
@@ -0,0 +1,38 @@
+const arrayToSum = (array) => {
+  let sum = 0
+  array.forEach((element) => (sum += element))
+  return sum
+}
+
+const arrayToNumbers = (array) => {
+  const datasheet = require('./datasheet.js').datasheet
+  const values = array.map((element) => datasheet[element])
```

Comment:
what does the .map and require command do for your code?

Reply:
It takes in the original array of letters and takes each letter and looks in the datasheet for the assigned value and then creates a new array of just the values.